src/canvas/canvas_styling.rs

```diff
@@ -54,6 +56,8 @@ impl Default for CanvasColours {
                 .bg(currently_selected_bg_colour),
             table_header_style: Style::default().fg(HIGHLIGHT_COLOUR),
             ram_style: Style::default().fg(FIRST_COLOUR),
+            #[cfg(not(target_os = "windows"))]
+            cache_style: Style::default().fg(FOURTH_COLOUR),
```

Comment:
In this file, you still need to add a setter in `set_colours_from_palette` calling the corresponding function (e.g. `set_swap_colour`).